valohai_yaml/objs/pipelines/execution_node.py

```diff
@@ -36,16 +36,16 @@ def lint(self, lint_result: LintResult, context: LintContext) -> None:
         if not step:
             lint_result.add_error(f'{error_prefix} does not exist')
             return
-        step_inputs = step.get_input_defaults()
-        step_parameters = step.get_parameter_defaults()
-        for input_name in self.get_input_defaults():
+        step_inputs = step.get_serialized_inputs()
+        step_parameters = step.get_serialized_parameters()
+        for input_name in self.get_serialized_inputs():
             if input_name not in step_inputs:
                 lint_result.add_error(f"{error_prefix}: input {input_name} does not exist in step")
-        for parameter_name in self.get_parameter_defaults():
+        for parameter_name in self.get_serialized_parameters():
             if parameter_name not in step_parameters:
                 lint_result.add_error(f"{error_prefix}: parameter {parameter_name} does not exist in step")
 
-    def get_parameter_defaults(self) -> Dict[str, Any]:
+    def get_serialized_parameters(self) -> Dict[str, Any]:
```

Comment:
I... don't see why this was renamed? Especially since this does return a parameter-to-default mapping, and not serialized `Parameter` objects, and even so it only returns those that have `"default"` set?

Reply:
@euvolen-valohai seems that this is implemented if so resolve the conversation :+1: same for other renamings.